spec/lib/evss/base_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require_dependency 'evss/claims_service'
+
+describe EVSS::BaseService do
+  context 'with an outage' do
+    let(:service) { EVSS::ClaimsService.new({}) }
```

Comment:
Could you create a dummy class that has the BaseService as its parent and instantiate that here? Then you could either `send :get/:post` on that instance or define methods in the dummy class that call those? That may not work with the way the breakers services are set up though. In the end I think what you had before may be the best, just in a different file 😃